server/src/main/java/org/candlepin/dto/api/v1/ImportUpstreamConsumerDTO.java

```diff
@@ -205,7 +205,17 @@ public ImportUpstreamConsumerDTO clone() {
         ImportUpstreamConsumerDTO copy = super.clone();
 
         ConsumerTypeDTO type = this.getConsumerType();
-        copy.setConsumerType(type != null ? (ConsumerTypeDTO) type.clone() : null);
+        if (type != null) {
+            copy.setConsumerType(new ConsumerTypeDTO()
```

Comment:
So, we don't get a clone method??? This looks tedious.

Reply:
Yes